packages/worldpayaccess-integration/project.json

```diff
@@ -0,0 +1,23 @@
+{
+    "root": "packages/worldpayaccess-integration",
+    "sourceRoot": "packages/worldpayaccess-integration/src",
+    "projectType": "library",
+    "targets": {
+        "lint": {
+            "executor": "@nrwl/linter:eslint",
+            "outputs": ["{options.outputFile}"],
+            "options": {
+                "lintFilePatterns": ["packages/worldpayaccess-integration/**/*.ts"]
+            }
+        },
+        "test": {
+            "executor": "@nrwl/jest:jest",
+            "outputs": ["coverage/packages/worldpayaccess-integration"],
+            "options": {
+                "jestConfig": "packages/worldpayaccess-integration/jest.config.js",
+                "passWithNoTests": true
```

Comment:
Just saw this, do we need this?

Reply:
I'll remove it, it was generated

Reply:
but all packages have this

Reply:
🤦 maybe need to go and update for all packages.